lib/Pipeline.js

```diff
@@ -83,6 +83,7 @@ Pipeline.processJSON = function(gltf, options) {
  * @param {Object} [options.compressTextureCoordinates=false] Flag to run compressTextureCoordinates stage.
  * @param {Object} [options.kmcOptions=undefined] Options to pass to the generateModelMaterialsCommon stage, if undefined, stage is not run.
  * @param {Object} [options.quantize] Flag to run quantizeAttributes stage.
+ * @param {Boolean} [options.mergeVertices = false} Flag to merge duplicate vertices, which can produce a smaller model size but greatly increases conversion time.
```

Comment:
I added doc for preserve and mention it in the doc for merge vertices.  I left the logic as-is.  This new option should hopefully go soon enough anyway, since it's just to work around #161